omnigibson/objects/dataset_object.py

```diff
@@ -103,6 +101,8 @@ def __init__(
         # Add info to load config
         load_config = dict() if load_config is None else load_config
         load_config["bounding_box"] = bounding_box
+        # All DatasetObjects should have xform properties pre-loaded
+        load_config["xform_props_pre_loaded"] = True
```

Comment:
what is this?

Reply:
I believe we avoid assigning xform properties for DatasetObject and OG assets robots for efficiency reasons. This is also a recent "intended" change from the previous multi env refactoring PR but it's not done correctly. My PR fixes it.